data/Viral kinetics of SARS-CoV-2/data.yaml

```diff
@@ -0,0 +1,105 @@
+analytes:
+  throatswab_SARSCoV2:
+    biomarker: SARS-CoV-2
+    description: SARS-CoV-2 RNA genome copy concentration in throat swab samples.
+      Specimens were collected from all patients at least 2 days after hospitalization
+      and physicians checked their symptoms and signs daily.
+    gene_target: unknown
+    limit_of_detection: unknown
+    limit_of_quantification: 1000
+    reference_event: PatientID
+    specimen: throat_swab
```

Comment:
In the caption of Figure 1, it says `nasal throat swab`. Maybe we should add `nasopharyngeal_swab` along with the `throat_swab` like:
```
    specimen:
    - nasopharyngeal_swab
    - throat_swab
```
Fiona's "kim2020viral" data has a similar example.